CHANGELOG.md

```diff
@@ -0,0 +1,129 @@
+# 📝 Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+---
+
+## [2.51.0] - 2025-11-01
+
+### 🔄 Major Migration Release
+
+This version represents a **complete migration** from [Light60-1/Microsoft-Rewards-Rewi](https://github.com/Light60-1/Microsoft-Rewards-Rewi) to [TheNetsky/Microsoft-Rewards-Script](https://github.com/TheNetsky/Microsoft-Rewards-Script).
```

Comment:
What does this sentence mean? Migration from what and how? The repo linked only has 1 commit in it. This doesn't make any sense.